scripts/formats/tokens.scss.hbs

```diff
@@ -0,0 +1,6 @@
+// generated by Theo (https://github.com/salesforce-ux/theo), edit `tokens.json` instead
+@mixin tokens() {
+{{#each props as |prop|}}
+  --ods-{{kebabcase prop.name}}: {{#eq prop.type "string"}}"{{/eq}}{{{prop.value}}}{{#eq prop.type "string"}}"{{/eq}};
```

Comment:
We might be able to optimise the `json` file and the templates by following our own convention, e.g.

```json
{
  "type": "border-radius",
  "name": "small",
  "value": "{!border-radius-small}"
},
{
  "type": "color",
  "name": "neutral-600",
  "value": "{!color-neutral-600}"
},
```

```hbs
{{#each props as |prop|}}
  --ods-{{prop.type}}-{{kebabcase prop.name}}: {{prop.value}};
{{/each}}
```

If `type` is a predefined enum, we could use `category` instead.
Regardless, I don't think the `eq string` bit is needed at all.

Reply:
Good suggestion, however I feel that because "value" would still need to alias to full key it might be just more confusing than having little more of code:

```json
{
  "type": "size",
  "category": "border-radius",
  "name": "small",
  "value": "{!border-radius-small}"
}
```

vs

```json
{
  "type": "size",
  "category": "border-radius",
  "name": "border-radius-small",
  "value": "{!border-radius-small}"
}
```

Reply:
Don't you think it's less confusing since a `theo alias` will be the full variable name, while a `theo prop` will be more or less equal to a Figma token?

```json
{
  "aliases": {
    "color-neutral-500": "180, 186, 197"
  },
  "props": [
    {
      "type": "color",
      "name": "neutral-500",
      "value": "{!color-neutral-500}"
    },
    {
      "type": "color",
      "name": "content-main",
      "value": "{!color-neutral-500}"
    }
  ]
}
```

i.e. `{!color-neutral-500}` has to point to an alias, because the prop is `color/neutral-500`

Reply:
OK, refactored this bit - have a look.

I can't use `type` here as type has to be "size" for border radius, this for web does not mean much but is used e.g. for Android generation.

Reply:
Looks good. I figured `type` was used some other way.
We might revert this decision in the future, but for now it looks way cleaner.